config/development.json

```diff
@@ -165,6 +165,7 @@
 		"p2-enabled": false,
 		"page/export": true,
 		"pattern-assembler/v2": true,
+		"pattern-library/readymade-templates": true,
```

Comment:
I added a different flag in https://github.com/Automattic/wp-calypso/pull/91980.

I don't have any opinions about which one to use, but would you mind removing one of them? 😄